.gitignore

```diff
@@ -4,7 +4,7 @@
 .idea/*
 !.idea/copyright
 .DS_Store
-/build
+build/
```

Comment:
nit: If you want to do this, then potentially think about deleting `app/.gitignore` since all that file has in it is `/build`.